modules/earthengine-layers/src/earth-engine-layer.js

```diff
@@ -24,13 +24,29 @@ const defaultProps = {
 
 export default class EarthEngineLayer extends CompositeLayer {
   initializeState() {
-    this.state = {};
+    this.state = {
+      frame: 0
+    };
+  }
+
+  _animate() {
+    const {
+      loopLength = 24, // unit corresponds to the timestamp in source data
```

Comment:
Can we discover this from data?

Reply:
Yes, since we're setting the pixel height/width. So `nFrames` = `imageBitmap.height / 256` which can be set as `loopLength`.

Reply:
ImageCollection maybe contains a way to get the number of images?

Reply:
I think that would require a server round trip, because I think that's lazily computed. But the above is possible _once you have the filmstrip image_.